freezefrog/__init__.py

```diff
@@ -1,17 +1,54 @@
-import calendar
 import datetime
+import time
+from unittest.mock import patch
 
 try:
-    from unittest.mock import patch
+    import pytz
+
+    PYTZ_SUPPORT = True
 except ImportError:
-    from mock import patch
+    PYTZ_SUPPORT = False
 
-__all__ = ['FreezeTime']
+__all__ = ["FreezeTime"]
 
 
 real_datetime = datetime.datetime
 
 
+def is_pytz(tz):
+    return hasattr(tz, "localize")
+
+
+def get_pytz_dst_from_fold(dt, tz, fold):
+    """
+    The relationship between pytz's `is_dst` and datetime's `fold` is not
+    obvious:
+
+    datetime's `fold` lets you say you want either the earlier time
+    (fold=0) or the later time (fold=1) in a DST.
```

Comment:
Maybe better? "during the DST transition period" instead of "in a DST".

Reply:
Done in https://github.com/closeio/freezefrog/pull/13/commits/5681cf629871344f5158cea1956d3afa33764190.